src/as_enum.rs

```diff
@@ -325,10 +334,11 @@ impl DrmModifier {
             consts::DRM_FOURCC_I915_Y_TILED_CCS => Self::I915_y_tiled_ccs,
             consts::DRM_FOURCC_I915_Y_TILED_GEN12_MC_CCS => Self::I915_y_tiled_gen12_mc_ccs,
             consts::DRM_FOURCC_I915_Y_TILED_GEN12_RC_CCS => Self::I915_y_tiled_gen12_rc_ccs,
+            consts::DRM_FOURCC_I915_Y_TILED_GEN12_RC_CCS_CC => Self::I915_y_tiled_gen12_rc_ccs_cc,
             x => Self::Unrecognized(x),
         }
     }
-    pub(crate) fn into_u64(self) -> u64 {
+    pub(crate) fn into_u64(&self) -> u64 {
```

Comment:
I believe some previous version of the generator changed this.

Reply:
This appears to be some internal function?

Reply:
Well that's confusing, it looks like it got added (and perhaps later updated) in https://github.com/danielzfranklin/drm-fourcc-rs/commit/bc532e05f78b631e398352685374d6d5f3ece464#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42R224 yet the changes didn't trickle down into generated code?

Perhaps this crate lacks CI that makes sure the generated result remains up-to-date (but depends on headers in CI).

Reply:
Anyway, since the non-borrowing variant seems more correct to me, and that was how it's used previously, I think the generator should be fixed instead?

Reply:
I do agree we should change the generator to not borrow since the type is Copy

Reply:
Feel free to pick that up in a separate PR, as well as add some very simple CI that errors out (see for example `git status --porcelain`) if anything changed?

Reply:
Made an issue for that #14

Reply:
And a pull request for the `&self` -> `self` (#15)